From 5a89c21de22813bd63e4744555057491b3460f5c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 19 Nov 2018 05:20:23 -0700
Subject: [PATCH 12/15] msm8974-common: sepolicy: Resolve dnsmasq denials

* avc: denied { getattr } for path="socket:[11154]" dev="sockfs"
  ino=11154 scontext=u:r:dnsmasq:s0 tcontext=u:r:netd:s0
  tclass=unix_stream_socket permissive=0

Change-Id: Ib4d9b6963812d07735259c5d4b2d672d1e763ed0
---
 sepolicy/common/dnsmasq.te | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 sepolicy/common/dnsmasq.te

diff --git a/sepolicy/common/dnsmasq.te b/sepolicy/common/dnsmasq.te
new file mode 100644
index 0000000..45b26c7
--- /dev/null
+++ b/sepolicy/common/dnsmasq.te
@@ -0,0 +1 @@
+allow dnsmasq netd:unix_stream_socket getattr;
-- 
2.17.1

